.chloggen/adx-update-go-lib.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
if the change is just an update to remove a deprecated library, it doesn't need a changelog entry

Reply:
Thanks. Will remove it.